torch_geometric/nn/aggr/attention.py

```diff
@@ -47,11 +47,13 @@ def reset_parameters(self):
 
     def forward(self, x: Tensor, index: Optional[Tensor] = None,
                 ptr: Optional[Tensor] = None, dim_size: Optional[int] = None,
-                dim: int = -2) -> Tensor:
+                dim: int = -2, batch: Optional[Tensor] = None,
```

Comment:
@rusty1s This allows the `batch` and `batch_size` arguments to be passed to the underlying MLP modules within the `AttentionalAggregation`. Since we provide the arguments to the `MLP` class itself, we must provide that same capability to the modules that use the `MLP`.